chamber/analysis/experiments.py

```diff
@@ -17,7 +17,9 @@
 - `results_from_csv` -- get analysis results from a .csv file.
 
 """
+from itertools import repeat
 import math
+from multiprocessing import cpu_count, Pool
```

Comment:
Please change this to `import multiprocessing as multi` and then explicitly call `multi.cpu_count` and `multi.Pool`.